tests/test_cli.py

```diff
@@ -171,6 +177,9 @@ def test_find_files_recursively(self):
              os.path.join(self.wd, 'dos.yml'),
              os.path.join(self.wd, 'empty.yml'),
              os.path.join(self.wd, 'no-yaml.json'),
+             os.path.join(self.wd, 'non-ascii/utf16be'),
+             os.path.join(self.wd, 'non-ascii/utf16le'),
+             os.path.join(self.wd, 'non-ascii/utf8'),
```

Comment:
For consistency with the path just below, and for easier `grep`ping, can you change to `utf-8`, `utf-16-le`, etc.? Thanks!